.github/workflows/testing.yml

```diff
@@ -0,0 +1,35 @@
+name: Testing
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  schedule:
+    # runs these tests every Sunday, at 12 UTC
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule
+    - cron:  '0 12 * * 0'
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
```

Comment:
We should add to the developer docs that we can put `[ci skip]`, etc. in commit messages to skip the CI stuff.